DESCRIPTION

```diff
@@ -45,10 +49,9 @@ Imports:
     whisker,
     fs,
     remotes,
-    stringi,
-    tm,
     knitr,
-    messydates
+    messydates,
```

Comment:
Wrong place for this comment, but doesn't qData also need messydates?

Reply:
qData does not import messydates. The package is imported here since `qCreate::standardise::dates()` is a wrapper for messydates.

Reply:
But doesn't qData need to know messydates too, to work with `messydt` class vectors?

Reply:
As `qData::consolidade()` develops/expands, messydates will likely become necessary to "resolve" date variables differently than other variables. For now, some of this has not yet been implemented and `messydt` objects are threated as character. Is there anything you would like me to do in relation to the messydates dependencies here for now? Thank you.